prime-router/src/main/kotlin/azure/ActionHistory.kt

```diff
@@ -519,7 +519,7 @@ class ActionHistory {
         jsonGen.writeStringField("service", orgReceiver.name)
         jsonGen.writeStringField(
             "sending_at",
-            if (sendingAt == null) "immediately" else "$sendingAt"
+            if (sendingAt == null) "never" else "$sendingAt"
```

Comment:
@jimduff-usds is this correct? I thought we could still send immediately? Does an immediate send now fill in the `sendingAt` property?

Reply:
I also think we should add additional context here instead of just saying "never," we should note that this was submitted with `skipSend` true. I know @jimduff-usds said we should just have "never" in the `sendingAt` field, but I propose it would be better and more clear to say `never - skipSend specified`. I'll leave it up to Jim to provide the final word on that.

Reply:
@jimduff-usds do you agree with the change Maurice is proposing ?

Reply:
Yes, I like Mo's idea, and his  `never - skipSend specified` is good.  Please change the ticket as well.

However, the bigger problem is that the ticket very clearly says, 
```
Do no harm check: PLEASE also carefully check that other cases without SkipSend, have not changed! For example, if the SkipSend is NOT used, the above scenario should still say "sending_at" : "immediately",, or it should specify a timestamp.
```

You'll need to set up a transport that has no batching/merging to test this.

Reply:
fixed